pkgs/build-support/dotnet/build-dotnet-module/fetch-deps.sh

```diff
@@ -1,7 +1,14 @@
 set -e
 
-tmp=$(mktemp -d)
-trap 'chmod -R +w "$tmp" && rm -fr "$tmp"' EXIT
```

Comment:
If I `cp -r /nix/store/[some package] foo && rm -rf foo`, I get 

```
rm: cannot remove 'foo/[...]': Permission denied
```

I believe it's when a file is in a readonly directory?